src/Compatibility/Directory.Packages.props

```diff
@@ -6,6 +6,7 @@
     <!-- We pin the code analysis version when not in source build as we need to support running on older
          SDKs when the OOB package is used. -->
     <PackageVersion Update="Microsoft.CodeAnalysis.CSharp" Version="4.4.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="$(MicrosoftCodeAnalysisWorkspacesCommonPackageVersion)" />
```

Comment:
Why add this next to a pinned version?  Seems like the stuff under this condition should all be pinned?

Reply:
Here are similar "latest" versions https://github.com/dotnet/sdk/blob/a076841e2ec09e26bab6facafba7412dbd8bcefd/Directory.Packages.props#L31

Reply:
Hmm good catch. I didn't have to put it there. This needs to be outside and unconditioned.

Reply:
Fixed @ericstj.

Reply:
I see - we're ok keeping this one specific to compatibility projects, I guess?

Reply:
Also, I guess we won't be using this new dependency from existing msbuild tasks -- otherwise we'd want to pin it for non-source-build just like we do for CodeAnalysis.

Reply:
Since no other project is consuming it at the moment. I thought it was better to keep it only for compat.

Reply:
> Also, I guess we won't be using this new dependency from existing msbuild tasks -- otherwise we'd want to pin it for non-source-build just like we do for CodeAnalysis.

Let me double check in the next PR. I'll pay close attention to where this new dependency will be consumed. I'll make sure to pin it if it ends up affecting directly or indiretly the GenAPI/ApiCompat MSBuild tasks.

Reply:
We should be careful _not_ to use it there, since it doesn't ship next to roslyn bits in the SDK and we wouldn't want to redist a different version next to our task.